Jenkinsfile

```diff
@@ -337,99 +346,101 @@ pipeline {
+        trainer.gpus=[0] \
+        +trainer.fast_dev_run=true \
+        model.use_cache=false \
+        model.language_model.pretrained_model_name=megatron-bert-345m-cased trainer.distributed_backend=null \
+        exp_manager.root_dir=exp_ner_megatron_bert_base_cased'
+        sh 'rm -rf examples/nlp/token_classification/exp_ner_megatron_bert_base_cased'
+      }
     }
 
-     stage('L2: NER with cased Megatron') {
-          when {
-            anyOf{
-              branch 'candidate'
-              changeRequest target: 'candidate'
-            }
-          }          
-          failFast true
-          steps {
-                sh 'cd examples/nlp/token_classification && \
-                python ner.py \
-                model.data_dir=/home/TestData/nlp/token_classification_punctuation/ \
-                trainer.gpus=[0] \
-                +trainer.fast_dev_run=true \
-                model.use_cache=false \
-                model.language_model.pretrained_model_name=megatron-bert-345m-cased trainer.distributed_backend=null \
-                exp_manager.root_dir=exp_ner_megatron_bert_base_cased'
-                sh 'rm -rf examples/nlp/token_classification/exp_ner_megatron_bert_base_cased'
-           }
+    stage('L2: NER with uncased Megatron') {
+      // Megatron uses all GPUs under the hood, cannot parallel
+      when {
+        anyOf{
+          branch 'candidate'
+          changeRequest target: 'candidate'
+        }
+      }
+      failFast true
+      steps {
+        sh 'cd examples/nlp/token_classification && \
+        python ner.py \
+        model.data_dir=/home/TestData/nlp/token_classification_punctuation/ \
+        trainer.gpus=[0] \
+        +trainer.fast_dev_run=true \
+        model.use_cache=false \
+        model.language_model.pretrained_model_name=megatron-bert-345m-uncased trainer.distributed_backend=null \
+        exp_manager.root_dir=exp_ner_megatron_bert_base_uncased'
+        sh 'rm -rf examples/nlp/token_classification/exp_ner_megatron_bert_base_uncased'
+      }
     }
 
-    stage('L2: NER with uncased Megatron') {
-          when {
-            anyOf{
-              branch 'candidate'
-              changeRequest target: 'candidate'
-            }
-          }          
-          failFast true
-	  steps {
-                sh 'cd examples/nlp/token_classification && \
-                python ner.py \
-                model.data_dir=/home/TestData/nlp/token_classification_punctuation/ \
-                trainer.gpus=[0] \
-                +trainer.fast_dev_run=true \
-                model.use_cache=false \
-                model.language_model.pretrained_model_name=megatron-bert-345m-uncased trainer.distributed_backend=null \
-                exp_manager.root_dir=exp_ner_megatron_bert_base_uncased'
-                sh 'rm -rf examples/nlp/token_classification/exp_ner_megatron_bert_base_uncased'
+    stage('L??: ASR Checkpoints tests') {
```

Comment:
@tkornuta-nvidia can you please remind us what L that should be?

Reply:
That is my problem with Ls, they are artificial.

Types of tests that I am sticking to - and that are reflected in markers:
unit, integration, system, acceptance (+additional: docs)

From our mapping: L2 and above = acceptance tests
https://docs.google.com/presentation/d/1i-aaCTCijBt9N_NPjlFcCYpDiCA48jYtZ22OIJPNLAY/edit#slide=id.g7de03c0e01_0_51